cms/static/sass/partials/_variables.scss

```diff
@@ -39,6 +39,11 @@ $f-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
 // ====================
 $transparent: rgba(0,0,0,0); // used when color value is needed for UI width/transitions but element is transparent
 
+// +Colors - UXPL new pattern library colors
+// ====================
+$primary-accent: rgb(14, 166, 236);
```

Comment:
Are these colors supposed to be the same as $uxpl-blue-base and $uxpl-blue-hover-active?